### DISCUSSION TITLE
OpenXR and WMR

### DISCUSSION THREAD

**Discussion by hemasree99s** [microsoft/MixedRealityToolkit-Unity#10469] [General]
Any macro is present to check if WMR is enabled or OpenXr is enabled?

**Comment by MaxWang-MS** [comment]
Hi @hemasree99s, please take a look at the properties in `XRSettingsUtilities`. Hope that helps.

**Reply by hemasree99s** [reply]
@MaxWang-MS XRSettingsUtilities is present inside Editor namespace, hence cannot use the methods present here to check if its WindowsXR/OpenXR enabled
